registration/backends/default/views.py

```diff
@@ -139,3 +140,18 @@ def activate(self, *args, **kwargs):
 
     def get_success_url(self, user):
         return ('registration_activation_complete', (), {})
+
+
+class ResendActivationView(BaseResendActivationView):
+    def resend_activation(self, *args, **kwargs):
+        """
+        Given an email, look up user by email and resend activation key if user
+        is not already activated or previous activation key has not expired.
+        """
+        site = get_current_site(self.request)
+        email = kwargs.get('email', '')
```

Comment:
`resend_activation_mail` looks for a RegistrationProfile with `email`. An empty string would probably not match any RegistrationProfiles, and the function returns False. This will show the resend_activation.html template. Is there a standard for returning bad request instead?